controllers/books.js

```diff
@@ -1,56 +1,77 @@
 const Book = require("../models/book");
 
-const NUMBER_OF_BOOKS_TO_SHOW = 3;
+const NUMBER_OF_BOOKS_TO_SHOW = 6;
 
 const BooksController = {
   Index: (req, res) => {
     const { query } = req;
-    console.log({ query });
 
     Book.find((err, books) => {
       if (err) {
         throw err;
       }
-      console.log({ books: books.slice(0, NUMBER_OF_BOOKS_TO_SHOW) });
+      const filteredBooks = books
+        .filter((filteredByDate) => filteredByDate.discussionDate != "TBC")
+        // This needs to sort by date, but currently passing in months as a string.
+        .sort((a, b) => { return b.discussionDate - a.discussionDate});
+
+      let nextButton = false;
+      if (filteredBooks.length > NUMBER_OF_BOOKS_TO_SHOW) {
+        nextButton = true;
+      }
```

Comment:
prefer const over let, if something is a constant you know it will be defined always and it won't be changed unexpectedly 
```suggestion
      const nextButton = filteredBooks.length > NUMBER_OF_BOOKS_TO_SHOW;
```
showNextButton is a more accurate var name

Reply:
Noted, thanks. I will review the variable naming.